config/reference_paths.config

```
@@ -1,21 +1,9 @@
-// reference params and files
-assembly         = 'Homo_sapiens.GRCh38.104'
+// path to reference files
+ref_json = "$projectDir/references/scpca-refs.json"
+ref_metadata = "$projectDir/references/ref-metadata.tsv"
```

Preserving spacing to make parallel structure easier to see. And brackets for consistency.
```suggestion
ref_json            = "${projectDir}/references/scpca-refs.json"
ref_metadata        = "${projectDir}/references/ref-metadata.tsv"
```